src/Viewer/renderers/solution.jsx

```diff
@@ -10,27 +30,27 @@ export default function Solution(props) {
     return null;
   }
 
-  let icon;
-  let childrenToRender = null;
-  let infoBlockStyle = { display: 'none' };
+  // let icon;
```

Comment:
Refactoring the js makes it harder for us to debug later and does not follow the logic used in the other doenetML renderers. Please revert these changes